catalog/spatial/registry/registry-federation-admin-service-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -25,16 +28,20 @@
     </bean>
 
     <bean id="refreshRegistryEntries"
-          class="org.codice.ddf.registry.federationadmin.service.impl.RefreshRegistryEntries">
+          class="org.codice.ddf.registry.federationadmin.service.impl.RefreshRegistryEntries"
+          destroy-method="destroy" init-method="init">
         <cm:managed-properties persistent-id="org.codice.ddf.registry.federationadmin.service.impl.RefreshRegistryEntries"
-                               update-strategy="container-managed" />
+                               update-strategy="container-managed"/>
         <property name="federationAdminService" ref="federationAdminService"/>
         <property name="filterBuilder" ref="filterBuilder"/>
         <property name="registryStores" ref="registryStore"/>
+        <property name="executor" ref="threadPool"/>
     </bean>
 
     <bean id="executor" class="java.util.concurrent.Executors" factory-method="newSingleThreadScheduledExecutor"/>
```

Comment:
Can we give better names to the `executor` and `threadPool` beans? It's difficult to tell which one is used for what, especially since `executor` is also used as a property name on L38.